google-cloud-clients/google-cloud-bigtable-admin/src/test/java/com/google/cloud/bigtable/admin/v2/BigtableInstanceAdminClientTest.java

```diff
@@ -176,13 +175,13 @@ public void testCreateInstance() {
             .setInstanceId(INSTANCE_NAME.getInstance())
             .setInstance(
                 com.google.bigtable.admin.v2.Instance.newBuilder()
-                    .setType(Type.DEVELOPMENT)
+                    .setType(com.google.bigtable.admin.v2.Instance.Type.DEVELOPMENT)
```

Comment:
StorageType.DEVELOPMENT.toProto()?

Reply:
Would rather keep it as is to keep the distinction between protos & wrappers clear